Tests/SentryTests/SentryCrash/SentryFrameInAppLogicTests.swift

```diff
@@ -0,0 +1,216 @@
+import XCTest
+
+class SentryFrameInAppLogicTests: XCTestCase {
+    
+    private class Fixture {
+        
+        func getSut(inAppIncludes: [String] = [], inAppExcludes: [String] = [] ) -> SentryFrameInAppLogic {
+            
+            return SentryFrameInAppLogic(
+                inAppIncludes: inAppIncludes,
+                inAppExcludes: inAppExcludes
+            )
+        }
+    }
+    
+    private let fixture = Fixture()
+    
+    func testNotInApp() {
+        XCTAssertFalse(fixture.getSut().is(inApp: "a/Bundle/Application/a"))
+        XCTAssertFalse(fixture.getSut().is(inApp: "a.app/"))
+    }
+    
+    func testInAppInclude() {
```

Comment:
L: this is a matter of preference, feel free to ignore it, the downside of adding multiple test cases within a single method is when it fails, you clearly don't know which one failed, so you have to either read the logs or debug it, my preference is small/named tests, but maybe because this is a common thing on Kotlin that you can describe pretty well, not sure about Swift.